src/pages/iou/IOUModal.js

```diff
@@ -103,7 +110,6 @@ class IOUModal extends Component {
         this.addParticipants = this.addParticipants.bind(this);
         this.createTransaction = this.createTransaction.bind(this);
         this.updateComment = this.updateComment.bind(this);
-        this.getReady = this.getReady.bind(this);
```

Comment:
`this.getReady` ??? what does it do? 😄

Reply:
[getReady](https://github.com/Expensify/Expensify.cash/blob/main/src/pages/iou/IOUModal.js#L154-L156) fetches the `preferredCurrency` whenever a new IOU is initiated.

Reply:
Yes, I can see. Just commenting on the name being hilariously bad 🙃

Reply:
Oh lol yeah 😆